transmart-core-db/grails-app/domain/org/transmartproject/db/metadata/DimensionDescription.groovy

```diff
@@ -100,6 +100,6 @@ class DimensionDescription {
      * @return a list of all dimensions
      */
     static List<Dimension> getAllDimensions() {
-        findAll { name != LEGACY_MARKER }*.dimension
+        findAll { name != DimensionDescription.LEGACY_MARKER }*.dimension
```

Comment:
Is this change needed here? For other methods from this class that are using LEGACY_MARKER we do not use this kind of self-reference